doc/api/typescript.md

```diff
@@ -52,7 +52,7 @@ added: v22.6.0
 
 > Stability: 1.1 - Active development
```

Comment:
```suggestion
> Stability: 1.2 - Release candidate
```
We should also add an entry in this document that says on what version type stripping was enabled by default